src/components/List/Cells/ApplicationNameCell.tsx

```diff
@@ -1,6 +1,15 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
+import { Header } from 'semantic-ui-react'
 import { CellProps } from '../../../utils/types'
 
-const ApplicationNameCell: React.FC<CellProps> = ({ application }) => <p>{application.name}</p>
+const ApplicationNameCell: React.FC<CellProps> = ({ application }) => (
+  <Header
+    size="small"
+    as={Link}
+    to={`/application/${application.serial || 0}`}
+    content={application.name as string}
+  />
```

Comment:
I wonder why there is no "hover" style for this link. Something to look into with styling...

Reply:
Sure. Can this be flagged as something to be done as part of #592?